### PR TITLE
Write about the SHA-2 family.

### PR DESCRIPTION
**Pull request by EdOverflow**
Here's the chapter on SHA-2. I am looking forward to your feedback and ideas.

### CODE REVIEW

File: Crypto101.bib [hunk 1]
```diff
@@ -327,4 +327,13 @@ @misc{cryptoeprint:2015:967
   howpublished = {Cryptology ePrint Archive, Report 2015/967},
   year = {2015},
   note = {\url{http://eprint.iacr.org/2015/967}}
+}
+@misc{eprint-2008-18172,
+  title={New Collision attacks Against Up To 24-step SHA-2},
```
Comment (lvh) [inline Crypto101.bib]: Please wrap SHA-2 in `{}`

**Comment by EdOverflow** [comment]
I have wrapped SHA-2 in `{}` and the URL in `\url`.

**Comment by lvh** [comment]
Sorry; I wasn't clear: it's _also_ necessary to replace the note such that the URL is in the note.

**Comment by lvh** [comment]
(You can get rid of the note currently there; it doesn't seem relevant)

**Comment by EdOverflow** [comment]
OK, no worries. ;)

**Comment by lvh** [comment]
Thanks!

**Pull request merged**